plugins/catalog-backend-module-bitbucket-server/api-report.md

```diff
@@ -67,6 +88,52 @@ export class BitbucketServerEntityProvider implements EntityProvider {
   refresh(logger: LoggerService): Promise<void>;
 }
 
+// @public (undocumented)
+export namespace BitbucketServerEvents {
```

Comment:
No need to export this right?

Reply:
@Rugvip If I try to internally export these (so give them the @internal tag as opposed to @public), I get the error:

```
Internal Error: Unable to analyze the export "BitbucketServerProject" in
/backstage/dist-types/plugins/catalog-backend-module-bitbucket-server/src/lib/types.d.ts

You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
```

If I don't export them, the other parts of the code can't access them. Any suggestions?

Reply:
@Rugvip Please check in on this when you get the chance.